vault/share/plugins/vault/20_cfe_basics/vault_lib.cf

```diff
@@ -38,6 +39,16 @@ bundle agent vault_fetch_config()
   vars:
       "config_path" string => "/var/rudder/plugin-resources/vault.json";
       "vault.config" data => readjson("${config_path}");
+ 
+      "vault.cert" string => "--insecure",
+      ifvarclass => strcmp("vault.config[verify_certificate]", "true"),
```

Comment:
no need for is variable here is is strcmp works it means the variale exists

Reply:
and `--insecure` means we do *not* verify the certificates, the logic is inverted here